src/codecs/lz4.ts

```diff
@@ -0,0 +1,12 @@
+import lz4 from 'lz4'
+import { Buffer } from 'buffer'
+
+export default {
+    async compress(encoder: { buffer: Buffer }) {
+        return lz4.encode(encoder.buffer)
```

Comment:
Kanskje vi kan få noen til å skru på lz4 i f.eks dev og se om dette funker? Sånn for sikkerhetsskyld. Men ikke kritisk for å merge